package.json

```diff
@@ -10,7 +10,8 @@
     "node-cmd": "^4.0.0",
     "open": "^7.3.0",
     "promptly": "^3.2.0",
-    "puppeteer": "^5.5.0"
+    "puppeteer": "^5.5.0",
+    "superagent": "^6.1.0"
```

Comment:
I'm a little skeptical about the need for another package. What about this package can we not accomplish with puppeteer?

Reply:
No js required for walmart, using superagent reduces resource usage on host machine and decreases response time

Reply:
Have you considered the resource usage of having to run an instance of two separate libraries? I have a hard time believing resource usage of two instances of puppeteer is somehow greater than one instance of puppeteer and another instance of a completely separate library in superagent.

Reply:
Each instance of chromium that puppeteer uses consumes anywhere from 5-10% CPU during launch and 50-60 MB of ram.  You can see the performance hit by watching the difference between setTimeout and the actual execution time of the search on PS Direct thread -- there's a 4-5 second hit to performance on an i9

superagent simplifies native node http request functionality with nearly 0 overhead.  On a previous project, I scraped somewhere to the tune of 1800 API calls in less than 3 seconds -- puppeteer wouldn't even have one incognito tab open and available in that timeframe

Reply:
@outphase84 Sounds good. You've clearly done your homework :)